docs/docsite/rst/user_guide/playbooks_filters.rst

```diff
@@ -1062,6 +1062,10 @@ To use one value on true and another on false (new in version 1.9)::
 
     {{ (name == "John") | ternary('Mr','Ms') }}
 
+To use one value on true, one value on false and a third value on null (new in version 2.8)::
+
+   {{ (name == "John") | ternary('Mr', 'Ms', omit) }}
```

Comment:
Could we come up with a more sensible example? It took me a while to figure out what the logic meant because it didn't make sense to me that I'd want to use `Mr` with `John`, `Ms` with any other non-null value (what about `Peter`?) and leave out the title if the person has no first name. What's a real-world situation where someone would use this logic?

Reply:
`(name is Male)|ternary('Mr', 'Ms', omit)`

Reply:
Male being a 'made up test'

Reply:
`. versionadded:: 2.8`

Reply:
addressed by 773987c74847800b39ec17236e52e27ee34fad5a